.circleci/config.yml

```diff
@@ -0,0 +1,36 @@
+version: 2.1
+orbs:
+  python: circleci/python@2.1.1
+
+jobs:
+  test:
+    parameters:
+      version:
+        default: "current"
```

Comment:
does "current" ever get used here? Should we default to the python version we're treating as LTS instead?

Reply:
Oh that's a good catch. It should never actually fallback to the default, but you're absolutely right in that `current` won't be correctly interpreted as a version.